crates/aptos/CHANGELOG.md

```diff
@@ -3,9 +3,15 @@
 All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-- Add safe methods to delete a profile, to rename a profile, and to output the private key of a profile.
-- Add `aptos update movefmt`. This installs / updates the `movefmt` binary, which is needed for the new `aptos move fmt` subcommand.
-- Integrate the Move formatter `movefmt` which is now available via `aptos move fmt`
+
```

Comment:
Will the updates to `crates/aptos/Cargo.toml` and `Cargo.lock` (for package "aptos") be coming in a different PR?

Usually, the version bumps and changelog are done in one PR, from what I have seen so far.

Reply:
Good catch, done.